[DO NOT MERGE] Revise cost management dashboard retirement details (#8178)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/cost-management/cost-management.md

# About cost management <Lifecycle status='private_preview,managed,managed_plus' />

> :::info Retiring the cost management dashboard on November 25, 2025

Author: ```suggestion
:::info Retiring the cost management dashboard
```

> We’re focusing on building a more scalable and integrated cost-insights experience, expected in early 2026.

Author: ```suggestion
On November 25, 2025, we'll be retiring the cost management dashboard. Instead, we'll focus on building a more scalable and integrated cost-insights experience, expected in early 2026.
```

> On November 25, 2025, we'll be retiring the cost management dashboard. Instead, we'll focus on building a more scalable and integrated cost-insights experience, expected in early 2026.

Reviewer: sugg more concise flow
```suggestion
On November 25, 2025, we'll be retiring the cost management dashboard to focus on building a more scalable and integrated cost-insights experience, expected in early 2026.
```

> In the meantime, you can analyze model-level costs using the [dbt Model Cost Calculator package](https://github.com/dbt-labs/dbt-pov-model-cost-calculator).

Author: ```suggestion
In the meantime, you can analyze model-level costs using the [dbt Model Cost Calculator package](https://github.com/dbt-labs/dbt-pov-model-cost-calculator). Reach out to your dbt Labs account team with any concerns or questions.
```

> In the meantime, you can analyze model-level costs using the [dbt Model Cost Calculator package](https://github.com/dbt-labs/dbt-pov-model-cost-calculator). Reach out to your dbt Labs account team with any concerns or questions.

Author: ```suggestion
In the meantime, you can analyze model-level costs using the [dbt Model Cost Calculator package](https://github.com/dbt-labs/dbt-pov-model-cost-calculator). Please reach out to your dbt Labs account team with any questions or concerns.
```
